### DISCUSSION TITLE
[Bug]: How to customize the name in the menu?

### DISCUSSION THREAD

**Discussion by baobaomi900901** [storybookjs/storybook#28932] [Help]
### Describe the bug

This is a way for me to write myself to reduce the workload.
<img width="952" alt="image" src="https://github.com/user-attachments/assets/e0d71826-edf6-45d2-ae2b-a1658e251e73">

But after using this method, the name of the menu on the left is not what I want to customize.
<img width="2347" alt="image" src="https://github.com/user-attachments/assets/32f6f013-12bf-4064-acff-70d20cf5529e">

I need your help. Thank you

### Reproduction link

https://stackblitz.com/edit/github-redeex?file=src%2Fhooks%2Findex.ts,src%2Fstories%2FbuttonEpx.vue,src%2Fstories%2FButton.stories.ts&preset=node

### System

```bash
"@storybook/addon-essentials": "^8.1.1",
    "@storybook/addon-interactions": "^8.1.1",
    "@storybook/addon-links": "^8.1.1",
    "@storybook/blocks": "^8.1.1",
    "@storybook/manager-api": "^8.1.2",
    "@storybook/test": "^8.1.1",
    "@storybook/theming": "^8.1.2",
    "@storybook/vue3": "^8.1.2",
    "@storybook/vue3-vite": "^8.1.1",
    "@vitejs/plugin-vue": "^5.0.4",
```

**Comment by vanessayuenn** [comment]
@baobaomi900901 Stories are statically analyzed due to performance reasons. The way you are dynamically generating stories isn't supported out of the box. If you really don't want to define each story manually, I suggest you take a look at [this RFC](https://github.com/storybookjs/storybook/discussions/23177) for the two alternatives we recommend.
